### DISCUSSION TITLE
Multiple docker instances with load balancing

### DISCUSSION THREAD

**Discussion by rilrom** [payloadcms/payload#6584] [Q&A] [answered]
Hi everyone.

Could someone please help me understand if there’s any issues with Payload running over multiple docker instances with load balancing? Does it need some sort of common storage location or shared cache? I’m using the custom server approach with Next.js.

Thanks!

**Comment by matthijs166** [comment, marked as answer]
Sounds like a cool project!

For the media storage, it's essential to use a shared location or cloud-based solution, such as the one offered by the [Payload CMS Cloud Storage Plugin](https://www.npmjs.com/package/@payloadcms/plugin-cloud-storage).

Additionally, ensure that all instances are connected to the same database to maintain consistency.

I don’t anticipate any other significant issues.
